src/Freya.Machine.Extensions.Http/Handlers.fs

```diff
@@ -96,7 +96,9 @@ let private description x =
      *> languages x.Languages
 
 let private data x =
-    Response.body_ %= (fun body -> body.Write (x, 0, x.Length); body)
+    freya {
+        let! body = Freya.Optic.get Response.body_
+        body.Write (x, 0, x.Length) }
```

Comment:
@kolektiv my attempt to fix machine writing for Suave. This does not appear to have worked, but I committed it b/c I understand this far better than the previous one-liner. Feel free to revert.